File: 01.마이크로서비스 설계 - MS Archi/02.구글 드라이브/googledrive_python/video processing/VideoProcessed.py
from AbstractEvent import AbstractEvent
import json
from datetime import datetime

class VideoProcessed(AbstractEvent):
    id : int
    fileId : int
    url : str
    
    def __init__(self):
        super().__init__()
        self.id = None
        self.fileId = None
        self.url = None

File: 01.마이크로서비스 설계 - MS Archi/02.구글 드라이브/googledrive_mybatis/video processing/src/main/java/google/drive/dto/event/VideoProcessed.java
package google.drive.dto.event;

import java.util.Date;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;

@JsonIgnoreProperties(ignoreUnknown = true)
@Data
public class VideoProcessed extends AbstractEvent {

    private Long id;
    private Long fileId;
    private String url;

    public VideoProcessed(){
        super();
    }
}